form/without_class.rst

```diff
@@ -103,6 +103,4 @@ but here's a short example:
     ``Default`` group when creating the form, or set the correct group on
     the constraint you are adding.
 
-.. code-block:: php
-
-    new NotBlank(array('groups' => array('create', 'update')))
+        new NotBlank(array('groups' => array('create', 'update')))
```

Comment:
You can remove the `.. code-block::` notation but you then need to use the default php one by adding a double colon `::` at the end of the line 104.